src/style.css

```diff
@@ -65,8 +78,36 @@ a {
 
 .button {
   background: transparent;
-  border: 3.5px solid #1f2328;
-  padding: 0 0.4rem;
+  border: 0.2em solid var(--clr-neon);
+  border-radius: 0.3em;
+  padding: 0.25em 0.4em;
+  cursor: pointer;
+  letter-spacing: 0.14em;
+  text-shadow: 0 0 0.125em hsl(0 0% 100% / 0.3), 0 0 0.12em var(--clr-neon);
+  box-shadow: inset 0 0 0.7em 0 var(--clr-neon), 0 0 0.7em 0 var(--clr-neon);
+  transition: background-color 100ms linear;
+}
+
+.button::after {
+  content: "";
+
+  /* position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0; */
```

Comment:
- [ ] Kindly remove this commented lines of code, this will make your codebase to be clean. 👍